CHANGELOG.md

```diff
@@ -108,6 +111,10 @@
 * Removed old error message that's superseded by `requires_dependencies`.
 * Removes using `hi_res` as the default strategy value for `partition_via_api` and `partition_multiple_via_api`
 
+### Enhancements
```

Comment:
I thought these were whitespace changes, but these changelog entries might have been duplicated due to a merge conflict or something like that. Can you remove these (L114-L117) since they don't pertain to this PR? Only changelog update should be the version and the entry describing this PR